src/test/java/org/zalando/pazuzu/FeatureApiTest.java

```diff
@@ -2,13 +2,9 @@
 
 import com.fasterxml.jackson.core.JsonProcessingException;
 import org.junit.Test;
-import org.springframework.http.HttpEntity;
-import org.springframework.http.HttpHeaders;
-import org.springframework.http.HttpMethod;
-import org.springframework.http.HttpStatus;
-import org.springframework.http.MediaType;
-import org.springframework.http.ResponseEntity;
+import org.springframework.http.*;
```

Comment:
No wildcard imports. (only if you have good reasons for it - 6 imports from same package is no good reason)